src/codegen/compile_classical.ml

```diff
@@ -2540,7 +2540,7 @@ module Closure = struct
       I32Type :: Lib.List.make n_args I32Type,
       FakeMultiVal.ty (Lib.List.make n_res I32Type))) in
     (* get the table index *)
-    Tagged.load_forwarding_pointer env ^^
+    (*Tagged.load_forwarding_pointer env ^^ FIXME: NOT needed, accessing immut slots*)
```

Comment:
there can never be a mutation of the `funptr_field` between old space and newspace. Maybe we should have `Tagged.load_immutable_field` (= `Tagged.load_field`) to signify that `load_forwarding_pointer` is not needed.